CONTRIBUTING.md

```diff
@@ -12,30 +15,30 @@
     - [SQL](#sql)
     - [Python](#python)
 
-# General Guidelines
+## General Guidelines
 
-* By contributing to LandSandBoat, either through issues or pull requests, you are expected to abide by the rules laid out here in this Contributing Guide.
-* We do not support out-of-date clients or client modification.
-* We do not support piracy of any kind. We encourage you to maintain an active retail subscription and support the game.
+- By contributing to LandSandBoat, either through issues or pull requests, you are expected to abide by the rules laid out here in this Contributing Guide.
+- We do not support out-of-date clients or client modification.
+- We do not support piracy of any kind. We encourage you to maintain an active retail subscription and support the game.
 
-# License
+## License
 
-* We operate under [GNU General Public License v3.0](https://github.com/LandSandBoat/server/blob/base/LICENSE). We do not accept contributions that use other more restrictive licenses (such as AGPL3).
+- We operate under [GNU General Public License v3.0](https://github.com/LandSandBoat/server/blob/base/LICENSE). We do not accept contributions that use other more restrictive licenses (such as AGPL3).
 
-# Workflow Guide
+## Workflow Guide
 
-* It is **always** better to come into Discord and ask a question instead of investing a lot of time in work that we're going to ask your to rewrite or split up.
-* Cite your sources. This can be comments in your code, or your commit messages. Pull Request descriptions and comments will get lost over time.
-* If you're commiting work on someone else's behalf, use git's `--author` argument so they get the credit they deserve.
-* Make your commit messages meaningful, or amend/rebase once you're ready to push.
+- It is **always** better to come into Discord and ask a question instead of investing a lot of time in work that we're going to ask your to rewrite or split up.
```

Comment:
we should update this to direct ppl to github discussions as we no longer have a project discord